storage/storage.cpp

```diff
@@ -716,7 +717,8 @@ void Storage::OnDownloadFinished(QueuedCountry const & queuedCountry, DownloadSt
     OnFinishDownloading();
   };
 
-  if (status == DownloadStatus::Completed && m_integrityValidationEnabled)
+  // FIXME: can't verify diff w/ hash of complete file before applying - downloads diff in loop on error
+  if (status == DownloadStatus::Completed && m_integrityValidationEnabled && fileType != MapFileType::Diff)
   {
     /// @todo Can/Should be combined with ApplyDiff routine when we will restore it.
     /// While this is simple and working solution, I think that Downloader component
```

Comment:
I think this is covered in [`Storage::ApplyDiff`](https://github.com/organicmaps/organicmaps/blob/b600aafa24db9baf629125ee8d9cb06609c5847a/storage/storage.cpp#L1419) but I wanted to check before disabling it.